### PR TITLE
Release 2.2.0

### PR DESCRIPTION
**Pull request by boyarkin-anton**

### CODE REVIEW

**Review by StanislavDevIOS — APPROVED** [review]

File: Adamant/Wallets/Lisk/LskWalletService.swift [hunk 1]
```diff
@@ -280,6 +296,32 @@ extension LskWalletService {
...
+            
+            group.wait()
+        } else {
+            netHash = Constants.Nethash.test
+            accountApi = Accounts(client: .testnet)
+            transactionApi = Transactions(client: .testnet)
+            nodeApi = LiskKit.Node(client: .testnet)
+        }
+        
+//        nodeApi.info { result in
```
Comment (adamant-al) [inline Adamant/Wallets/Lisk/LskWalletService.swift:317]: Do we need this commented code?

File: Adamant/Wallets/TransferViewControllerBase+QR.swift [hunk 2]
```diff
@@ -19,7 +19,14 @@ extension TransferViewControllerBase {
         switch AVCaptureDevice.authorizationStatus(for: .video) {
         case .authorized:
             qrReader.modalPresentationStyle = .overFullScreen
-            present(qrReader, animated: true, completion: nil)
+           // present(qrReader, animated: true, completion: nil)
```
Comment (adamant-al) [inline Adamant/Wallets/TransferViewControllerBase+QR.swift:22]: Do we need this commented code?

File: Adamant/Wallets/TransferViewControllerBase+QR.swift [hunk 3]
```diff
@@ -56,7 +70,14 @@ extension TransferViewControllerBase {
             
             alert.addAction(UIAlertAction(title: String.adamantLocalized.alert.cancel, style: .cancel, handler: nil))
             alert.modalPresentationStyle = .overFullScreen
-            present(alert, animated: true, completion: nil)
+           // present(alert, animated: true, completion: nil)
```
Comment (adamant-al) [inline Adamant/Wallets/TransferViewControllerBase+QR.swift:73]: Same here

File: AdamantShared/AdamantResources.swift [hunk 4]
```diff
@@ -87,7 +91,7 @@ struct AdamantResources {
     //    static let ethereumExplorerAddress = "https://ropsten.etherscan.io/tx/" // Testnet
     
     // MARK: LSK
-    static let liskExplorerAddress = "https://explorer.lisk.io/tx/"
+    static let liskExplorerAddress = "https://liskscan.com/transaction/" //"https://explorer.lisk.io/tx/"
```
Comment (adamant-al) [inline AdamantShared/AdamantResources.swift:94]: Remove commented https://explorer.lisk.io/tx/ and https://testnet-explorer.lisk.io/tx

File: LiskKit/Sources/Core/APIClient.swift [hunk 5]
```diff
@@ -166,8 +176,13 @@ public struct APIClient {
         }
 
         guard let result = try? JSONDecoder().decode(R.self, from: data) else {
-            let error = try? JSONDecoder().decode(APIError.self, from: data)
-            return .error(response: error ?? .unknown)
+            if let error = try? JSONDecoder().decode(APIError.self, from: data) {
+                return .error(response: error)
+            }else if let error = try? JSONDecoder().decode(APIErrors.self, from: data),
```
Comment (adamant-al) [inline LiskKit/Sources/Core/APIClient.swift:181]: Do we have linter attached?
"}else" — no space here

File: LiskKit/Sources/Core/APINode.swift [hunk 6]
```diff
@@ -34,14 +34,24 @@ extension Array where Element == APINode {
...
+
+    public struct Service {
+        public static let mainnet: [APINode] = [
+            .init(origin: "https://service.lisk.io")
+        ]
+        public static let testnet: [APINode] = [
+            .init(origin: "https://testnet-service.lisk.io")
+        ]
+    }
+
```
Comment (adamant-al) [inline LiskKit/Sources/Core/APINode.swift:54]: We must not connect third-party nodes in our app

**Pull request merged**